modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -158,15 +158,19 @@ exports.saveOAuthUserProfile = function (req, providerUserProfile, done) {
               lastName: providerUserProfile.lastName,
               username: availableUsername,
               displayName: providerUserProfile.displayName,
-              email: providerUserProfile.email,
               profileImageURL: providerUserProfile.profileImageURL,
               provider: providerUserProfile.provider,
               providerData: providerUserProfile.providerData
             });
+              
+            // Email intentionally added later to allow defaults (sparse settings) to be applid.
```

Comment:
This part of the comment is a little confusing. We're basically bypassing the constructor so that we **dont** apply the default value for the Email, which happens to be an empty string. You can mention that our intention is to omit the Email field from the User collection document when the provider doesn't send us back an email, activating the sparse option on that field.